spring-boot-admin-client/src/main/java/de/codecentric/boot/admin/client/config/ClientRuntimeHints.java

```diff
@@ -42,7 +44,9 @@ private static void registerReflectionHints(RuntimeHints hints) {
 					MemberCategory.INVOKE_PUBLIC_CONSTRUCTORS)
 			.registerConstructor(Application.Builder.class.getDeclaredConstructor(), ExecutableMode.INVOKE)
 			.registerMethod(Application.Builder.class.getMethod("build"), ExecutableMode.INVOKE)
-			.registerMethod(Application.class.getMethod("builder"), ExecutableMode.INVOKE);
+			.registerMethod(Application.class.getMethod("builder"), ExecutableMode.INVOKE)
+			.registerMethod(DefaultApplicationFactory.class.getMethod("onWebServerInitialized",
```

Comment:
This is necessary due to the use of `@EventListener`, which is meta-annotated with `@Reflective` since the underlying implementation invokes the annotated method using reflection.